### DISCUSSION TITLE
Run selection/line in VS

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7666] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Visual-Studio-2022:-how-to-run-selected-/10401436)._

---
When I tried to run select code it is not possible. I am using python. I selected the piece of code to run, I click right button, I selected "Run Code" and instead of the piece of code the entire program runs.
Could you help me?
Thanks in advance.

---
#### Feedback Bot on 6/27/2023, 05:47 PM: 

(private comment, text removed)

---
### Original Solutions
(no solutions)

**Comment by StellaHuang95** [comment]
Run selection/line is not a feature we currently support as far as I know, but agree that it would be nice to have. Moving this issue to discussion as an enhancement request for comments and upvotes.
